proxy/http2/Http2SessionAccept.cc

```diff
@@ -57,6 +57,10 @@ Http2SessionAccept::accept(NetVConnection *netvc, MIOBuffer *iobuf, IOBufferRead
 
   Http2ClientSession *new_session = THREAD_ALLOC_INIT(http2ClientSessionAllocator, this_ethread());
   new_session->acl_record         = session_acl_record;
+  new_session->host_res_style     = ats_host_res_from(client_ip->sa_family, options.host_res_preference);
+  new_session->outbound_ip4       = options.outbound_ip4;
+  new_session->outbound_ip6       = options.outbound_ip6;
+  new_session->outbound_port      = options.outbound_port;
```

Comment:
Curious, why not just have a "new_session->options = options" ?

Reply:
Historical inertia would be my guess. When I first started mucking with this I had no idea what I was doing, so I followed the existing copies. At this point, I'd lean more toward having the NetVC track the Accept object and not copy at all. It would be a useful clean up project for a mentored beginner.

Reply:
Make it so #1. File an Issue so we don't forget ?

Reply:
Stumbled across this while merging in this change locally. Didn't see an issue for this, so created #3489.